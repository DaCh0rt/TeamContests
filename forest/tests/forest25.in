992 487
343 -3020
5467 6210
1008 4747
901 5824
-7548 -3321
-5087 -7789
-8650 -5812
-1868 9945
-4396 5093
-8628 -6232
8213 -6040
7940 1896
-8260 9145
-3884 7900
-4675 -1418
-1394 -633
-1446 5520
-188 -2351
5310 -6257
-2060 5607
7155 2936
7731 -2414
4000 -2507
-8700 -1035
2854 4055
5085 8359
2295 -5114
-8474 1162
8631 6623
-6130 -9851
4880 5605
9819 -5513
2661 -8946
4839 -2871
613 -7975
-5086 2740
5656 5364
-6231 9586
-1453 955
-5763 1005
4300 3272
4362 684
7975 -43
6880 4075
5139 -7318
-7585 -6253
-1134 -2973
-5759 2427
-6702 -7305
4246 -5688
-6083 6653
-6069 9514
4602 7724
-482 -1795
7174 1370
-3969 -9957
5961 -9559
-5462 7201
-8194 -4493
-1075 9859
4987 8678
6804 -4324
-5479 7111
-6103 -243
-5773 -310
-9916 -903
-7031 9358
-4198 -823
6040 3834
8645 -6784
5266 -9701
-1766 -964
7362 4502
3216 -7347
-4947 4944
326 -9149
6665 -8380
2203 7214
-9233 966
-2696 -6372
-6005 -8841
-3506 -8114
-2247 1001
-2482 9896
-7660 -6210
4184 1063
-8117 -9834
-7923 1454
7296 3783
-918 -106
-2987 7385
8302 5121
-2600 3266
3785 3639
1897 -631
7215 7646
-8447 2534
-331 4427
-5794 505
-4698 -7561
-4303 -9350
5240 -8712
-5659 -1267
7880 -922
-1616 4623
-2428 -3532
-7666 3890
5966 -9738
-6054 2587
-2388 2335
3550 6242
5792 -3647
-2774 3216
259 8420
-6093 -4913
2307 -846
-1623 -6044
5221 9234
-4511 7324
4093 8143
9189 -9105
-5369 -1237
-357 9359
4170 547
4680 -1929
4923 -5621
4816 -460
-5065 3830
-1329 9779
4284 3487
3480 -7867
4095 8866
1818 -9709
328 -8565
7860 467
1088 507
-9488 8962
-6853 3058
-8437 -6528
-8367 -275
-3525 -8242
-1641 8075
2975 -4598
-1482 -6888
-5629 -4529
-5994 -5117
-8076 -5964
590 2128
4009 -1520
508 -1710
7367 -2224
7874 -2689
7191 7630
9321 -5709
4115 -8582
-6569 -7217
700 -8199
1542 -4920
5602 -9383
5688 4298
5785 -8036
-5791 7325
4292 6363
-1695 9516
5619 -1706
3454 -6754
-3700 5213
8746 8077
-7625 -5668
3030 4605
6464 2619
2707 -9248
-1405 -4762
-7042 1279
9441 1761
6551 -5535
-7048 9427
7773 -160
-6019 6408
-6443 -6016
2870 7781
7777 7372
9689 3616
-9084 8553
-888 2482
7438 -1874
-1202 -8175
6383 1988
-1245 1227
5398 -6270
-968 -6096
-3608 5489
8802 -2248
3312 1970
4386 2515
9185 8214
9307 -4455
7168 7867
-7810 2271
3947 7317
7860 6720
-892 3166
-5886 2781
5205 -2341
6293 7297
-8812 -1750
-3289 3651
-9954 6693
-6633 5173
2733 3872
-4800 -464
6421 3058
2143 -7374
1833 -6511
8154 -1886
4132 -4512
71 1625
2473 -1306
-773 1854
577 -8408
8244 9178
-9573 3951
6998 -1612
3582 9555
7887 7895
-9290 5904
-5058 -1831
-3295 634
-5379 -730
-5910 6767
-4997 -9148
2937 8770
1255 8073
-8737 -6152
9824 -7479
8831 -4259
-5670 -1299
178 -5875
-8354 104
-9216 -3003
6512 -2944
3949 6056
-2858 4255
-2847 -83
-6469 7966
-1767 -2976
-6889 1816
-929 7591
8053 2402
-4121 -2669
8327 4038
-3858 -2873
-2712 5996
-2492 4925
5781 6127
-7420 -7712
6718 1507
-7049 -4578
-750 -6936
-3975 8003
4310 -2937
9415 -1496
7434 2681
2228 -7887
9396 41
2586 9171
-1059 -2510
8613 5800
6012 9930
6265 4137
5301 6383
-3953 8630
-7922 -3967
1104 3339
-2529 4174
6137 3937
8951 9540
-2601 3305
7829 7120
3230 3278
-4046 7403
3389 2136
-2620 7723
6369 8236
-737 -248
-6573 3890
2832 6940
-4035 5291
8912 -2084
-1 -2210
9759 6444
-4013 772
-28 -5731
5411 -8042
6992 -5095
748 -6700
-2824 1184
7015 -1108
4217 -5099
3217 4876
6233 1036
-7373 6886
3180 -8095
8535 2952
-1338 8277
4697 -8608
2303 -3736
8462 -4427
9912 -6098
-8946 3850
-6883 -6441
859 9790
-9114 -3322
-199 3653
3286 6879
-5763 -6478
7837 8133
-6682 9362
3906 -1547
-7786 -6972
5713 -687
3164 591
-1181 1354
5770 5124
-722 9233
6460 -3552
9001 6973
-9946 7699
-3180 -9081
2859 -4058
-2512 3842
-907 1108
8281 -8304
-1639 -9032
-5557 -6442
5928 -1502
7012 -2022
-4274 4895
-8986 2057
-263 5676
-9079 4859
2709 -791
-6856 5207
-8220 -3914
-1169 9736
-8997 -9420
3562 6885
9864 -4484
9278 7241
-3161 -4415
5005 -919
5207 1763
-1923 5550
9777 -1685
-712 -5737
-4025 -9741
2218 9351
-259 -7207
-3767 -5775
1500 5363
-8461 -4299
-1018 -589
3379 6597
8132 7053
8935 2141
-5625 -1643
8195 -1420
-7626 -2454
-8836 1886
-7816 528
9070 -1860
340 4342
8963 -6851
20 -9469
-3839 -8217
-1864 -2239
7933 -861
5511 2004
8848 8281
8942 -7666
-454 2703
-9857 -9865
1960 -7577
-6912 8216
-3428 4572
1428 -7730
5413 -7571
-6725 6353
8314 5404
-4827 -2206
-9960 2922
-6549 -9747
5681 4233
-6489 1102
-7900 -1984
1572 2077
1295 -3180
-4829 9364
5032 -8001
5768 -3947
-9963 3892
886 -4942
-8946 -166
7034 -4719
7275 4015
5807 9453
-3889 -1391
3261 -4297
-575 2963
-3145 -4539
-340 3668
3860 -9121
-4863 5691
4305 7306
50 -946
6678 5657
-4596 6948
-8042 -2727
1027 9544
-4895 -9672
-391 -2017
-2067 6546
-1525 -8414
-5618 -9281
1762 826
7238 9308
-2143 -5744
8357 2255
9975 1833
2218 1012
4670 2330
4229 -8924
-4273 -8883
3364 7288
-1428 3113
-6202 7217
-411 3156
3336 -2359
6020 4424
-5564 -444
-4752 5597
-3486 4900
-1288 1082
3563 -6771
4577 -1639
-7393 -567
-586 9432
1351 6317
5753 -9312
9357 -3668
6107 8142
591 6256
-2213 -5698
3258 -9357
-2020 -3639
5025 5783
8457 -7911
6823 -9192
-6361 -5361
6756 2174
-5862 -466
6143 8573
8868 3088
-9645 8923
5379 -3187
-2059 7263
-1525 -6289
-4215 -6924
9693 -6770
5135 1401
47 -8302
3669 5736
6196 -7717
-7085 5011
-9015 -7796
1361 3748
-1880 373
7587 -9654
3026 1534
-2093 5594
-5268 -5322
-8274 -6166
780 -9437
-2884 6924
-2301 8492
-2492 9129
-9240 -661
-4680 4152
-5923 -5349
-3877 5254
-376 3908
1088 3342
-7182 -3983
-8817 5779
5090 -7280
7505 -5481
-7403 7992
6626 7355
9635 -8149
8658 8912
4829 -5303
188 8725
9018 -5129
-5530 -5655
-4749 -1951
-7792 9726
-4887 -5727
-1344 -4446
-2349 -5440
-2955 1440
-2151 3126
-1744 9716
-7204 -3576
-7607 9664
-6126 -9518
-1525 -2876
-4360 6551
2751 -4662
-9049 -4569
4635 5909
5941 -7414
2477 -3279
1089 -393
6142 -3921
9971 -6061
-2980 2827
-1157 -3832
2878 9404
1956 -5252
8962 3834
6630 6779
-2712 3349
2712 6019
8349 -8232
1977 4235
-3438 4855
4804 -1833
1233 -4865
4337 4991
-992 7388
8143 -1013
-7988 7950
1591 5124
-9719 2334
-6996 8444
-4981 -4194
9251 9706
-2584 1602
-144 -2104
4721 9765
9410 2444
763 -5020
-2216 2561
-3119 -3130
349 -1878
-1202 6710
4456 6221
5952 -3501
5694 9763
-2103 -4424
-6231 -9891
-2832 8114
9828 4801
2744 9060
-6414 -8821
-370 2065
-9185 8678
-1338 -8609
8504 -8523
-7662 4399
-5538 3640
2278 -5905
-4724 -3248
-1676 4984
8709 -5914
6676 4717
-9766 -3921
6096 -8101
2738 -7390
-3601 -3509
5088 -1986
-3847 -8872
6210 -6680
6235 2163
8956 -4668
731 -146
747 -2379
5240 -7574
-8381 2074
-8859 5195
-8870 -9029
-5906 -7410
1413 -6514
-2824 -9617
6233 -561
7469 6573
3906 8144
-7635 -2132
2880 -259
-2267 -9043
7189 -2424
5408 -5622
-4552 4263
-9775 -3095
8982 795
1895 -6471
-4000 2562
-6733 -6982
-480 8029
-8162 -8623
9069 -5565
-5976 4369
9490 4219
-1663 -6752
2692 9136
-9130 7925
6658 6481
3434 7570
4131 8154
4419 -7149
-2861 -8306
112 4513
3322 4672
-2010 -4458
8330 1265
-9452 8372
7439 -8574
9121 2383
-3825 -8873
9496 6446
-4277 3361
-145 8002
-1898 -1400
-419 8921
-8530 9222
-3463 304
-5360 -7249
8754 5762
2004 -761
3476 -65
-4132 -6840
-3093 -6632
9729 -9973
2192 -7553
-5322 -10
8354 -8720
-7197 1197
-7687 -2507
-2563 -9138
-2004 -4600
5608 3625
2338 2664
8528 -1973
-6290 6488
6951 1230
8964 4258
-1865 -8239
108 -785
-6624 5199
8473 -6998
-4355 1969
-7784 4193
6325 3032
-4995 4974
-2709 -8119
6283 -1453
9377 640
-8905 5732
-9773 -6894
-4522 -3287
2141 -1012
-3723 -8230
3984 -3281
4450 -1802
5473 -2650
-3726 9157
2529 -3028
6526 -3035
1181 -8531
-1011 446
233 213
-3164 -2511
-6579 8882
2928 4473
-1423 8414
2336 -2298
-6555 -1300
610 -4007
-9309 -1893
-4383 6462
-7777 -3483
5074 9214
3042 -6712
9025 9488
9055 -567
-3503 2402
3476 -6039
5573 5967
-1418 4025
-1574 4665
-1112 -8392
4472 -31
3602 5205
4782 5084
-5369 5821
-290 9353
185 1628
6940 9905
6496 729
-2927 7650
-8173 8303
-1949 -1716
404 -8654
-5002 -7877
-5464 1215
4423 204
4276 1867
-8262 5696
4327 5432
-1814 -6732
8490 5269
-3001 -6926
4853 -8926
9216 -6580
9960 2326
-8216 -2762
-1409 9325
6862 9113
-9722 -3214
4273 -5298
6426 -8524
982 -5686
-2334 -4623
7680 7041
-8203 5911
1751 4749
-1642 2393
3704 -6073
3409 -6043
3527 9707
-1022 -9681
-4305 1922
8406 -3868
7491 8530
-2169 -9925
-8394 -7995
9068 798
-5178 -3609
8206 3256
6685 -975
1398 7959
-5714 3524
-3382 -8679
2752 -7921
-8045 1736
-1827 5477
-3038 7431
9256 -7457
-8453 -6367
-9915 1147
3026 9022
-4870 -6296
-9753 -1052
133 -2011
-2509 -4187
1305 -2399
4929 5914
-3643 2466
-7074 8055
-6757 -1666
-3817 8560
-1052 -3111
4834 1028
4947 -1949
7674 3523
4085 -649
3751 3122
4193 7965
2027 3082
7086 -1616
-8532 -1527
-5532 -2709
6451 356
8925 -672
-4148 5109
9201 -8602
-5227 1509
-6491 9308
8782 5060
-6379 -6196
-6108 -3637
3898 -7317
8469 -5067
-4081 206
9028 -3197
-9412 3947
-6243 1774
5385 6429
1346 7900
6788 7950
7361 -9669
-1208 -4893
-6101 7097
-2997 1337
-2050 7716
-6393 -7052
-9775 3611
-6517 8303
2471 4440
-812 6857
5712 -6101
4230 2467
-5962 -30
-4762 5324
-8060 422
-2876 4416
-6643 2933
-7070 -639
1566 -5496
-1334 446
6797 -3198
-9237 -9167
-3848 5646
1399 -4140
-7984 -94
3694 -679
2056 -3248
-7586 2318
2984 4502
-6821 2113
7029 -5355
6489 5599
9361 1016
-172 5835
4539 8329
-6262 5446
9537 -7439
8008 -4236
-4187 1604
6394 -4591
6639 5515
-8603 -8103
-8536 -9133
-774 7893
-8099 -5025
-822 1304
7476 3429
-8414 -2725
6434 -831
8461 -3650
-8846 -3144
4837 -3985
7803 -2819
3326 7631
8707 -893
-4874 6571
-959 7018
-6675 4605
-5952 7697
9514 -9212
-6387 7004
-2018 2244
9644 -6770
1720 -4414
8227 703
9241 -2318
-5817 6257
-1244 -1134
-1613 -9224
-1474 -7689
-1769 1213
3580 -1725
-3907 -4541
604 7552
-6992 4024
1641 -3685
-4241 -6897
-7467 -3065
-5742 -511
-2824 -9754
9393 -8462
8268 6980
861 9812
9304 1062
-7713 5164
-4766 5565
6793 -2392
1031 1809
-3939 -3535
5332 -4461
-2481 -2269
-8508 -5098
-4661 5793
1432 4946
-8589 3121
742 -7556
8114 2156
3692 -1471
2374 -577
-7769 3757
3352 -3913
-544 2558
9361 -3451
4920 -9299
-6336 370
7551 -677
8536 4369
-6436 -45
7690 -5432
2822 -7627
2820 6003
5863 5748
-6809 -5892
-3891 -1069
9246 -502
6607 2338
-4589 -6081
-5684 -4551
-2795 -5802
-7488 -8079
1680 -2090
216 9101
5165 2542
1701 7316
16 4543
4683 5897
-7010 3804
9153 -6746
-257 -1248
9873 -4032
-2266 1228
-5089 9298
5253 -7373
9664 -3669
4826 638
-6575 8782
2024 9773
4501 -8764
-1316 8648
-1197 7412
-9100 364
-8626 -3646
1048 -8597
7739 7721
6518 7936
3516 8356
-3197 -1478
4229 -5832
-417 8263
-2860 -105
2436 -9690
-5461 -336
-4386 9974
-3279 4623
-1658 1337
2698 -5940
-3247 -6587
232 -5836
847 6830
5636 -258
8327 -3312
-9791 -2351
-7533 2693
5464 -3107
1528 3940
3751 -7485
-1068 3435
3823 1331
-5387 -890
-4418 700
-662 -3821
-8567 662
-7010 -8654
7142 5421
-9865 8621
9161 -7887
-7296 -327
-8824 715
-164 8941
-3688 -9679
-1425 -884
-3605 -9201
3375 -6166
170 5162
4189 8772
614 7128
2573 7661
2338 -3025
4858 -8666
-4480 -8694
-3999 -6008
1219 -8142
