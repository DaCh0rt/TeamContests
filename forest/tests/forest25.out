80321.73 429406968.28
